include/faabric/util/queue.h

```diff
@@ -14,14 +14,6 @@ class QueueTimeoutException : public faabric::util::FaabricException
     {}
 };
 
-class ExecutorFinishedException : public faabric::util::FaabricException
-{
-  public:
-    explicit ExecutorFinishedException(std::string message)
-      : FaabricException(std::move(message))
-    {}
-};
```

Comment:
This exception type is no longer handled in the scheduler, and only used in `mpi_native`. We can probably find a better way to do this when we come to resurrecting that part of the codebase.